about/index.markdown

```diff
@@ -75,6 +75,10 @@ Please use the following citation for MoveIt!
 
 [PickNik](http://picknik.ai/) was founded with the mission of growing the MoveIt! project through community building and contract consulting.
 
+<a href="https://acutronicrobotics.com"><img src="{{ site.url }}/assets/images/sponsors/acutronicrobotics.jpg" height="60" style="margin-top:20px"/></a>
+
+[Acutronic](https://acutronicrobotics.com/) sponsored a collaborative effort in 2019 to begin the port of MoveIt to ROS 2.0.
```

Comment:
Remove the first line and instead add a [Learn more] at end of sentence with link to corresponding blog post, please

Reply:
See my fixup commit below.